### ISSUE TITLE
Two issues in QML

### ISSUE THREAD

**Issue by rosly** [Yubico/yubikey-manager-qt#178]
1) Getting following error when clicking on "generate" button in "static password" form.
qml: qrc:///qml/OtpStaticPasswordView.qml:39: Error: Cannot assign QJSValue to QString

2) Getting following error when sending SIGINT to application:
```
"PyOtherSide error: Traceback (most recent call last):

  File \"qrc:///py/yubikey.py\", line 42, in wrapped
    return json.dumps(f(*args, **kwargs))

  File \"/usr/lib/python3.6/json/__init__.py\", line 231, in dumps
    return _default_encoder.encode(obj)

  File \"/usr/lib/python3.6/json/encoder.py\", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)

  File \"/usr/lib/python3.6/json/encoder.py\", line 257, in iterencode
    return _iterencode(o, 0)

KeyboardInterrupt"
qml: Return value of PyObject call is NULL: Traceback (most recent call last):

  File "qrc:///py/yubikey.py", line 42, in wrapped
    return json.dumps(f(*args, **kwargs))

  File "/usr/lib/python3.6/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)

  File "/usr/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)

  File "/usr/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)

KeyboardInterrupt

qml: qrc:///qml/YubiKey.qml:252: Error: Cannot assign [undefined] to int
```

System: Linux Ubuntu 
Compilation: From source master tip e58d79098c28bea7b138523cacb00aec214f622d

**Comment by dagheyman** [comment]
Thanks for the report, can reproduce the static password error. Expect a fix in the next version.

**Comment by emlun** [comment]
Fixed on master: Static password error was fixed in #184 (ff28bee217eb394b5ee7d7ebdfb6bc26f40ed6e8), signal handling was fixed in 512166a56df26eec4262bab48e017643cea7d21a and #217 (f2be9b610ec2c2f4db6339533dbb86db1a83b3d3).

**Comment by rosly** [comment]
Thanks!

wt., 17 gru 2019 o 17:39 Emil Lundberg <notifications@github.com>
napisał(a):
